dashboard/app/helpers/levels_helper.rb

```diff
@@ -74,6 +74,8 @@ def set_channel
   end
 
   def select_and_track_autoplay_video
+    return if @level.try(:never_autoplay_video) == "true"
```

Comment:
Incidentally, it looks like this has happened before with another Blockly attribute.

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/studio.rb#L351